src/Components/Aspire.MongoDB.Driver/AspireMongoDbDriverExtensions.cs

```diff
@@ -0,0 +1,174 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire;
+using Aspire.MongoDB.Driver;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Diagnostics.HealthChecks;
+using MongoDB.Driver;
+
+namespace Microsoft.Extensions.Hosting;
+
+/// <summary>
+/// Extension methods for connecting MongoDB database with MongoDB.Driver client.
+/// </summary>
+public static class AspireMongoDbDriverExtensions
+{
+    private const string DefaultConfigSectionName = "Aspire:MongoDB";
```

Comment:
```suggestion
    private const string DefaultConfigSectionName = "Aspire:MongoDB:Driver";
```

We've been following the component name in the config section name.